app/helpers/subscriptions_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module SubscriptionsHelper
+  def subscription_status_info(subscription)
```

Comment:
If you create a component for this status (fe. `SubscriptionStatusComponent`), we can place this helper logic and the view that uses it close together (and we can test it more easily)